include/arch/riscv/arch/model/statedata.h

```diff
@@ -35,8 +37,46 @@ NODE_STATE_END(archNodeState);
 
 extern asid_pool_t *riscvKSASIDTable[BIT(asidHighBits)];
 
+/* Kernel Page Tables */
+extern pte_t kernel_Level1_pageTable[BIT(PT_INDEX_BITS)] ALIGN(BIT(seL4_PageTableBits));
```

Comment:
Note: I'm mostly just pattern matching from the arm/../statedata.h here:
This should have the VISIBLE attribute, and I don't think that ALIGN has any effect on the extern declaration.

Reply:
Good to know. I'll add the `VISIBLE` tag and remove the alignment in the declaration